cmd/lambda/accounts/delete.go

```diff
@@ -1,93 +1,97 @@
 package main
 
 import (
-	"fmt"
-	"log"
 	"net/http"
 
-	"github.com/Optum/dce/pkg/rolemanager"
-	"github.com/aws/aws-sdk-go/service/iam"
-
-	"github.com/Optum/dce/pkg/api/response"
-	"github.com/Optum/dce/pkg/common"
-	"github.com/Optum/dce/pkg/db"
+	"github.com/Optum/dce/pkg/account"
+	"github.com/Optum/dce/pkg/data"
 	"github.com/gorilla/mux"
 )
 
 // DeleteAccount - Deletes the account
 func DeleteAccount(w http.ResponseWriter, r *http.Request) {
 
 	accountID := mux.Vars(r)["accountId"]
-	deletedAccount, err := Dao.DeleteAccount(accountID)
+	var dao *data.Account
+	if err := Services.Config.GetService(dao); err != nil {
+		ErrorHandler(w, err)
+		return
+	}
+	account, err := account.GetAccountByID(accountID, dao)
```

Comment:
Usually, when you're injecting an interface for DI it's much more common than not to have the injected arg be the first arg in the method. Consider instead account.GetAccountByID(dao, accountID). I'll also make note on the actual data.Account method.